CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ CHANGELOG
 
 ### Bots
 - `collectors.mail.collector_mail_url`: Fix bug which prevented marking emails seen due to disconnects from server (#852).
+- `parsers.spamhaus.parser_cert`: Handle/ignore 'AS?' in feed (#1111)
+
+### Packaging
+- Support building for more distributions
+- Use LSB-paths (/etc/intelmq/, /var/lib/intelmq/, /run/intelmq/) (#470)
+- Use quilt instead of native
```

Comment:
why?

Reply:
Because intelmq is not a native debian program. native does mean in the debian packagers context that it's only developed for debian. Will change it to "changed debian package format from native to quilt"